### ISSUE TITLE
title only displayed on mobile?

### ISSUE THREAD

**Issue by theLinkResolver** [ualbertalib/discovery#1122]
At least on my phone (Chrome via Android), when I am looking at a record, I only see the title, place hold links, and items table. None of the bib fields. Is this related to #1115?

**Comment by redlibrarian** [comment]
It's not related to the printing problem. Natasha and I made the decision to simplify the record view, but I can't remember the details. I will check with her when she gets back. One issue here is that we can't display *all* the bibliographic information in the mobile view; we  either have to exclude some or build some collapsible sections.

**Comment by theLinkResolver** [comment]
Yes, something had to be done about the mobile display, especially with long contents notes!

Of course it leads to new problems, like trying to figure out which of these is for the regular periodical vs. the buying guide:
![cr](https://user-images.githubusercontent.com/14913579/29681716-d32f9760-88c5-11e7-9a22-94168f35de79.png)
